DOKS-wordpress/README.md

```diff
@@ -761,25 +763,30 @@ Now, open a web browser and navigate to [localhost:9150/metrics](http://127.0.0.
 
-- [MonsterInsights](https://wordpress.org/plugins/google-analytics-for-wordpress/): is the best Google Analytics plugin for WordPress. It allows you to “properly” connect your website with Google Analytics, so you can see exactly how people find and use your website.
+**Note:**
+The wordpress admin password configured in the `values.yaml` of the Wordpress helm chart fails when attempting to log into WordPress admin. To change the password you need to connect to the the database and reset it. Follow this [article](https://docs.digitalocean.com/products/databases/mysql/how-to/connect/) on how to connect to a MySQL Database Cluster. Follow this [article](https://wordpress.org/support/article/resetting-your-password/) on how to change the password.
 
-- [All in One SEO](https://wordpress.org/plugins/all-in-one-seo-pack/): helps you get more visitors from search engines to your website. While WordPress is SEO friendly out of the box, there is so much more you can do to increase your website traffic using SEO best practices.
+Next you will configure the NitroPack plugin:
 
-- [SeedProd](https://wordpress.org/plugins/coming-soon/): is the best drag and drop page builder for WordPress. It allows you to easily customize your website design and create custom page layouts without writing any code.
+1. Navigate to the admin section of your WordPress instalation by going to: <https://YOUR_WORDPRESS_DOMAIN_HERE/wp-admin> and login with your wordpress admin credentials.
+2. Click on the `Plugins` menu item and `Add New` sub-menu.
+3. Search for the `Nitropack` plugin and from the results page click on the `Install Now` button. After the installation is complete, click on the `Activate` button. You should now see the plugin added added to your list of plugins
+4. Click on the `Settings` link under the plugin name. From this page click on the `Connect to NitroPack` button. This will let you log in or create an account with NitroPack.
+5. The NitroPack.io dashboard page should be opened with information related to the plan, optimized pages etc.
 
-- [LiteSpeed Cache](https://wordpress.org/plugins/litespeed-cache/): is an all-in-one site acceleration plugin, featuring an exclusive server-level cache and a collection of optimization feature
+Next you need to connect your site to NitroPack by following thses steps:
 
-- [UpdraftPlus](https://wordpress.org/plugins/updraftplus/): simplifies backups and restoration.  Backup your files and database backups into the cloud and restore with a single click.
+1. Navigate to [NitroPack](https://nitropack.io/) and log in with the account you created when configuring the plugin
+2. Click on the `Add new website` menu item, add the `Website URL`, `Website name`, click on the `Free subscription` option and click on the `Proceed` button
+3. If your domain is hosted on Clouflare you will be prompted to connect your `Cloudflare` account with the `NitroPack` account
```

Comment:
Don't forget to add a dot at the end of each sentence from the list of steps.

Reply:
Will fix this in the next commit